src/Runtime/ServiceManager.php

```diff
@@ -452,7 +452,7 @@ public function inflect(DrushContainer $container, $object): void
             $object->setConfig($container->get('config'));
         }
         if ($object instanceof LoggerAwareInterface) {
```

Comment:
OK, I guess we can't check `$object->logger()` here as I suggested unless we make `logger()` public. Did not consider impact of that, I think that method is defined in another project.

Reply:
That method is in Drush so we can change it but only for Drush 13 due to BC. I'll open a new PR for that, and fixup this one for Drush 12.

Reply:
I went the other way around. The Drush 12.x PR is at #5980. This PR will be for 13.x